app/scripts/views/cookies_disabled.js

```diff
@@ -18,26 +19,22 @@ function (Cocktail, BaseView, ConfigLoader, AuthErrors, Template, BackMixin) {
       BaseView.call(this, options);
 
       this._configLoader = new ConfigLoader();
+      this._storage = options.storage || Storage;
```

Comment:
Can you capitalize the `S` in both `this._storage` and `options.storage`? The lowercase convention indicates an instance, here we are using the module itself.